web/templates/scheduled_message.hbs

```diff
@@ -13,7 +13,7 @@
                     <span class="stream_topic_separator"><i class="zulip-icon zulip-icon-chevron-right"></i></span>
                     <span class="stream_topic">
                         <div class="message_label_clickable narrows_by_topic">
-                            {{topic}}
+                            <span>{{topic}}</span>
```

Comment:
It seems you're wrapping new spans around elements in a number of cases; can you talk about how this fixes the problem?

Reply:
In some case to not worsen the readability, I've just added spans

Reply:
OK, that makes sense.

Reply:
But I think in some of these cases, we should just use `{{~topic_name~}}` aka handlebars whitespace control.